### ISSUE TITLE
Building drivers crashes on missing translations

### ISSUE THREAD

**Issue by afbjorklund** [kubernetes/minikube#4909]
When trying to build either of the machine drivers, the build fails due to missing translations:

```Makefile
.PHONY: drivers
drivers: out/docker-machine-driver-hyperkit out/docker-machine-driver-kvm2
```

```
# k8s.io/minikube/pkg/minikube/translate
pkg/minikube/translate/translate.go:77:12: undefined: Asset
```

It requires this file to be generated (from JSON):
`pkg/minikube/translate/translations.go`

These files should not be needed nor used in the drivers, that are supposed to be lowlevel...

i.e. I don't think that we should have translation support, in the docker machine drivers code

**Comment by afbjorklund** [comment]
@medyagh @sharifelgamal anyone knows what is going here ?
Is `go build` flipping out because of using modules, or what ?

``` console
$ go build -v k8s.io/minikube/cmd/drivers/kvm
k8s.io/minikube/pkg/minikube/translate
# k8s.io/minikube/pkg/minikube/translate
pkg/minikube/translate/translate.go:77:12: undefined: Asset
```

is there a good tool to show dependencies ?

**Comment by afbjorklund** [comment]
Using `go list -json` is not so bad, it shows all the dependencies:

`GOOS=darwin go list -json cmd/drivers/hyperkit/main.go`

`GOOS=linux go list -json cmd/drivers/kvm/main.go`



It is the usage of either of these two imports:

* `k8s.io/minikube/pkg/minikube/config`

* `k8s.io/minikube/pkg/util`

**Comment by afbjorklund** [comment]
I added the translations as a dependency, but don't really like it.

It is definitely going to block any upstreaming... 😢 #3939 #3169

**Comment by josedonizetti** [comment]
@afbjorklund I've just tried to build the kvm driver on master and it worked without problem with both `make out/docker-machine-driver-kvm2` and `go build -v k8s.io/minikube/cmd/drivers/kvm`.
Is this error related to the link PR?

**Comment by afbjorklund** [comment]
It is related so far as I started building the drivers clean, and noticed the issue when the files are gone.

**Comment by josedonizetti** [comment]
Yeah, it fails. Probably related to https://github.com/kubernetes/minikube/blob/master/pkg/drivers/kvm/kvm.go#L28-L30

**Comment by afbjorklund** [comment]
The "constants" are OK, they're just hiding some strings. Will be a problem when upstreaming, though.

**Comment by josedonizetti** [comment]
@afbjorklund Agree. But constants are really simple to extract. Currently, the driver is using a function that depends on the translations directly  `util.CalculateSizeInMB(...)`:
https://github.com/kubernetes/minikube/blob/master/pkg/drivers/kvm/kvm.go#L107-L108

**Comment by afbjorklund** [comment]
> Currently, the driver is using a function that depends on the translations directly `util.CalculateSizeInMB(...)`:

Yeah, someone desperately wanted to tag a "mb" onto the default value...

The original driver only had these default as CLI flags, not in the driver code itself.
https://github.com/dhiltgen/docker-machine-kvm/blob/master/kvm.go#L102_L116

I can't really see why parsing the "millibytes" should be locale-specific, though.
(or why we can't spell MB, but maybe that is for our Windows friends or something)

But I think I will just leave it (for now), otherwise someone will want "[mebibytes](https://en.wikipedia.org/wiki/Mebibyte)".

**Comment by afbjorklund** [comment]
Fixed in ea4aeafa9def77b1e6efc66f99613220f172ca71
